code/game/gamemodes/wizard/magic_tarot.dm

```diff
@@ -654,6 +688,7 @@
 /datum/tarot/reversed/the_high_priestess
 	name = "II - The High Priestess?"
 	desc = "Run."
+	extended_desc = "summons Bubblegum to tear portals open around the user, that she will use to grab at them, and people around them at random, causing major damage." // iirc bubblegum is a she
```

Comment:
```suggestion
	extended_desc = "summons Bubblegum to tear portals open around the user that she will use to grab at them and people around them at random causing major damage." // iirc bubblegum is a she
```
incorrect comma usage. If you're worried about this being a run-on, consider splitting it into several sentences.

Reply:
restructured the sentence a bit